### ISSUE TITLE
spf31 (pombe or human)

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#1632]
from
https://github.com/pombase/curation/issues/1622#issuecomment-316205914

Does anyone know where the name spf31 comes from?
http://pombase2.aska.gen.nz/gene/SPBC1734.05c
This implicates in splicing but I can't find the evidence. In fact the human ortholog is annotated to splicing  by Reactome, but I cant trace the evidence there either
http://www.reactome.org/content/detail/R-HSA-53703
I can't find any papers in PubMed with spf31 (pombe or human) and I'm not sure if the human was named from pombe, or vice versa.
maybe @deustp01  can point me to the  source for  the Reactome annotation.

**Comment by Antonialock** [comment]
human...
PMID:9731529
PMID:12234937

**Comment by Antonialock** [comment]
(can't access nature at the moment...think it's been down all week)

**Comment by ValWood** [comment]
I've gone with NAS for now, I'm sure it is a splicing factor. I can check later and see if there is any EXP annotations at UniProt, and it might be published in pombe by then...

**Comment by deustp01** [comment]
Missed the request for information yesterday. Looking at our central database, I see this reference cited for the composition of the spliceosomal A complex: PMID:16809785. This is probably as close as we got to evidence that DNAJC8 (the apparent human counterpart of spf31) is part of that complex. I doubt the paper describes a function for it, though. And the splicing reaction itself (R-HSA-72160.5) is annotated in a way that allows us to avoid specifying the details of the splicing mechanism, e.g., what roles the various participating proteins play. My best guess, without re-reading the cited papers and looking at new data since 2003 when we annotated the reaction, is that we don't know. I see, though, that we relied on Adrian Kramer at CSHL for expert advice, and he certainly knows about splicing so perhaps he could be persuaded to comment.
